x-pack/plugins/security_solution/public/timelines/components/timeline/properties/helpers.tsx

```diff
@@ -84,49 +83,29 @@ SmallNotesButton.displayName = 'SmallNotesButton';
 interface NotesButtonProps {
   ariaLabel?: string;
   isDisabled?: boolean;
-  showNotes: boolean;
-  toggleShowNotes: () => void | ((eventId: string) => void);
+  toggleShowNotes?: () => void | ((eventId: string) => void);
   toolTip?: string;
   timelineType: TimelineTypeLiteral;
   eventId?: string;
   /**
-   * Number of notes associated with the event.
-   * Defaults to 0
+   * Number of notes. If > 0, then a red dot is shown in the top right corner of the icon.
    */
   notesCount?: number;
 }
 
 export const NotesButton = React.memo<NotesButtonProps>(
```

Comment:
nice simplification here but I think we could push it even a step further. If you look at the only place where this component is used, we always pass down a tooltip... this component is never used without a tooltip. So we could just make the `toolTip` property mandatory instead of optional, and we wouldn't even have to to `content={toolTip || ''}` but just `content={toolTip}`

Thoughts?

Reply:
You are right. Empty tooltip does not make sense... Either it should be `content = {tooltip || 'Some default tooltip'` 

or if tooltip is empty, `EuiTooltip` should not even be used.

Reply:
Done here: fae0f4a3c21